iphone/Classes/TiUIWebView.m

```diff
@@ -540,6 +540,13 @@ -(void)setHandlePlatformUrl_:(id)arg
     willHandleUrl = [TiUtils boolValue:arg];
 }
 
+-(void)setUserAgent_:(id)args
+{
+    ENSURE_TYPE(args, NSString);
+    NSDictionary *dict = @{@"UserAgent":args};
```

Comment:
Please allow the ability to pass `null` so the user-agent will be resetted to it's default,